pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [tool.poetry]
 name = "pandas-stubs"
-version = "1.4.3.220829"
+# Use the big version number so in CI, it will use the built wheel
+version = "1.4.3.999999"
```

Comment:
Why is that needed? To invalidate the cache?

This should already pick the newest wheel:
https://github.com/pandas-dev/pandas-stubs/blob/5b70b46007c90e9cd76d9169e4f9d9dcd038d218/scripts/test/run.py#L49

Reply:
The problem here is that if I build and install the wheel from the run script, it gets installed into the wrong python environment.  So now with `test.yaml` doing a direct build of the wheel and installation, I have to force it to install the newly built wheel, which requires a higher version number than what is available on pypi.

Reply:
> which requires a higher version number than what is available on pypi.

So pip is preferring the pypi.org wheel over the local wheel? There must be an option to ignore the remote wheel.

Could also simply bump the version number after each release so that the CI works but as soon as we forget it pip will install the wrong pandas-stubs - seems a bit fragile.

Reply:
Issue here is that we need this to work for PR's, which means that contributors have to change the version number.

In the `test.yaml`, I am now doing this:
```
python -m pip install --find-links=./dist --force-reinstall pandas-stubs
```
The `--find-links=./dist` makes it so that pip will look in the `dist` directory.  On Windows, you can't do `pip install dist/*`, so this is the workaround for that.